README.md

```diff
@@ -577,11 +578,16 @@ Previous releases may also have been signed with one of the following GPG keys:
 Information on the current Node.js Working Groups can be found in the
 [TSC repository](https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md).
 
+### Contributing to Node.js
+
+* [Contributing to the project][]
+* IRC (node core development): [#node-dev on chat.freenode.net][]
```

Comment:
Worth it to ditch the IRC link and just simply say "See CONTRIBUTING.md for details", since that venue is definitely mentioned there?

Reply:
+1 to @maclover7.
Not that the IRC is too noisy, but RTFMing before asking there will IMHO get the requester better answers.